src/Livewire/Settings/TicketTypeEdit.php

```diff
@@ -69,67 +65,47 @@ public function render(): View
 
     public function show(array $ticketType = []): void
     {
-        $this->ticketType = $ticketType ?:
-            array_fill_keys(
-                array_keys(resolve_static(CreateTicketTypeRuleset::class, 'getRules')),
-                null
-            );
-
-        unset($this->ticketType['uuid']);
-
-        $this->isNew = ! array_key_exists('id', $this->ticketType);
-
-        $this->ticketType['roles'] = $this->isNew ? [] : resolve_static(TicketType::class, 'query')
-            ->join('role_ticket_type AS rtt', 'ticket_types.id', '=', 'rtt.ticket_type_id')
-            ->whereKey($this->ticketType['id'])
-            ->pluck('rtt.role_id')
-            ->toArray();
+        $this->ticketType->reset();
+        $this->ticketType->fill($ticketType);
+
+        $this->isNew = ! $this->ticketType->id;
+
+        $this->ticketType->roles = $this->isNew
+            ? []
+            : resolve_static(TicketType::class, 'query')
+                ->join('role_ticket_type AS rtt', 'ticket_types.id', '=', 'rtt.ticket_type_id')
+                ->whereKey($this->ticketType->id)
+                ->pluck('rtt.role_id')
+                ->toArray();
     }
 
+    #[Renderless]
     public function save(): void
     {
-        if (($this->isNew && ! user_can('api.ticket-types.{id}.post')) ||
-            (! $this->isNew && ! user_can('api.ticket-types.{id}.put'))
-        ) {
-            $this->notification()->error(
-                __('insufficient permissions'),
-                __('You have not the rights to modify this record')
-            );
+        try {
+            $this->ticketType->save();
+        } catch (ValidationException|UnauthorizedException $e) {
+            exception_to_notifications($e, $this);
 
             return;
         }
 
-        $validated = $this->validate();
-
-        $ticketTypeService = app(TicketTypeService::class);
-        $response = $ticketTypeService->{$this->isNew ? 'create' : 'update'}($validated['ticketType']);
-
-        if (! $this->isNew && $response['status'] > 299) {
-            $this->notification()->error(
-                implode(',', array_keys($response['errors'])),
-                implode(', ', Arr::dot($response['errors']))
-            );
-
-            return;
-        }
-
-        $this->notification()->success(__('Ticket type saved successful.'));
-
-        $ticketType = $this->isNew ? $response->toArray() : $response['data']->toArray();
-
-        $this->skipRender();
-        $this->dispatch('closeModal', $ticketType)->to('settings.ticket-types');
+        $this->notification()->success(__('Ticket Type saved successful.'));
+        $this->dispatch('closeModal', $this->ticketType)->to('settings.ticket-types');
     }
 
+    #[Renderless]
     public function delete(): void
     {
-        if (! resolve_static(DeleteTicketType::class, 'canPerformAction', [false])) {
+        $ticketType = [$this->ticketType->id];
```

Comment:
array must be ['id' => $this->ticketType->id].
In the receiving method of the dispatch $data['id'] is used.